src/components/PopUpSample.jsx

```diff
@@ -19,16 +15,17 @@ class PopUpSample extends React.Component {
   }
 
   toggle() {
-    if (this.state.opened) {
-      console.log('close');
+    const { opened } = this.state;
+
+    if (opened) {
       this.setState({ opened: false });
```

Comment:
Change this and the following few lines to 
```
this.setState({ opened: !opened });
```
To remove duplicated code

Reply:
This sample pop up is written by liby! Zhe duan daima wo kan mengbi le!